src/Data/Geometry/Structure/Feature.hs

```diff
@@ -136,6 +169,10 @@ smartProcessGeometryInput n defVals input = case input of
     SJIExtended gi -> parsedFeatureCollection
                           { pfcSRID = newSRID
                           , pfcLonLatAlt = newLonLatAlt
+                          , pfcBlockColor = sjBlockColor gi
+                          , pfcActiveColor = sjActiveColor gi
+                          , pfcStaticColor = sjStaticColor gi
+                          , pfcLineColor = sjLineColor gi
```

Comment:
I propose to create a separate simple ADT with all scenario properties. Then use this type in parsedFeatureCollection and in City. Otherwise the number of properties will grow to much, and changing FeatureCollection according to it every time is not good.